src/templates/stickyboard-simple/package.json

```diff
@@ -45,7 +45,8 @@
         "request": "^2.88.2",
         "styled-components": "^4.4.1",
         "tls": "0.0.1",
-        "underscore": "^1.10.2"
+        "underscore": "^1.10.2",
```

Comment:
Where is the 'underscore' used?

Reply:
underscore is used in ./src/components/base/DrawerMenu.js, but I don't know why updated it.